packages/fcl/src/discovery/services/authn.js

```diff
@@ -78,9 +78,47 @@ const HANDLERS = {
 
 const spawnProviders = () => spawn(HANDLERS, SERVICE_ACTOR_KEYS.AUTHN)
 
+
+/**
+ * @typedef {Object} Service
+ * @property {string} f_type - A type identifier used internally by FCL
+ * @property {string} f_vsn - FCL protocol version
```

Comment:
Are any of these properties enumberable?

Reply:
Nope, why?

Reply:
Just curious.